guides/common/modules/proc_enable-insights-with-capsule.adoc

```diff
@@ -0,0 +1,25 @@
+[id='register-capsule-with-insights_{context}']
+
+= Register :SmartProxyServer: with Red Hat Insights
+
+You can use Red Hat Insights to proactively diagnose systems and downtime related to security exploits, performance degradation and stability failures.
+You can use the dashboard to quickly identify key risks to stability, security, or performance. 
+You can sort by category, view details of the impact and resolution, and then determine what systems are affected.
+
+.Prerequisites
+
+. Red Hat {Project} should be registered to Red Hat Customer Portal.
+
+.Solution
+
+. To register {Project} from Insights:

```

Comment:
from Insights would imply that the action happens in Insights but it looks like you're performing the action in Satellite? To register {Project} with Insights?